tests/CommonBuilderTest.php

```diff
@@ -100,8 +100,8 @@ public function isSanitizedDataProvider(): array
                 '',
             ],
             '#046' => [
-                '<img src="/typo3.org/logo.svg" alt="logo" loading="lazy" width="100" height="100" sizes="33.3vw" name="logo" align="left" border="0">',
-                '<img src="/typo3.org/logo.svg" alt="logo" loading="lazy" width="100" height="100" sizes="33.3vw" name="logo" align="left" border="0">',
+                '<img src="/typo3.org/logo.svg" alt="logo" loading="lazy" fetchPriority="low" decoding="async" width="100" height="100" sizes="33.3vw" name="logo" align="left" border="0">',
```

Comment:
seems to be used lowercased as HTML attr
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-fetchpriority

Reply:
this was my first solution but "Simon Schaufelberger" was complaining and did not response on my hint ; https://review.typo3.org/c/Packages/TYPO3.CMS/+/74926

Reply:
erledigt via https://review.typo3.org/c/Packages/TYPO3.CMS/+/74926/4..5